lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GSMobileIdentity.cpp

```diff
@@ -190,9 +200,9 @@ int M5GSMobileIdentityMsg::DecodeImsiMobileIdentityMsg(
           '\0';
     }
 
-    memcpy(&imsi->ciphertext, buffer + decoded, CIPHERTEXT_LENGTH);
-    decoded += CIPHERTEXT_LENGTH;
-    imsi->ciphertext[CIPHERTEXT_LENGTH] = '\0';
+    memcpy(&imsi->ciphertext, buffer + decoded, cipherTextLen);
```

Comment:
Is it possible to use bstring. Kindly check M5GProtocolConfigurationOptions.cpp for the conversion.

Reply:
Done